proposed/links.md

```diff
@@ -99,6 +99,24 @@ tool.  Some hypermedia formats support templated links while others do not, and
 have a special way to denote that a link is a template.  A Serializer for a format 
 that does not support URI Templates MUST ignore any templated Links it encounters.
 
+## 1.5 Evolvable collections
+
+In some cases, a Link Collection may need the ability to hbave additional links
+added to it. In others, a link collection is necessarily read-only, with links
+derived at runtime from some other data source. For that reason, modifiable collections
+are a secondary interface that may optionally be implemented.
+
+Additionally, some Link Collection objects, such as PSR-7 Response objects, are
+by design immutable.  That means methods to add links to them in-place would be
+incompatible.
+
+## 1.6 Evolvable link objects
+
+Link objects are in most cases value objects. As such, allowing them to evolve
+in the same fashion as PSR-7 value objects is a useful option. For that reason,
+an additional EvolvableLinkInterface is included that provides immutable modifiers
+using the same pattern as PSR-7.
```

Comment:
Describe what "the same pattern as PSR-7" means, so folks do not need to reference that document as well.